env.go

```diff
@@ -17,3 +22,26 @@ func (e *Env) Getenv(key string, def ...string) string {
 
 	return val
 }
+
+// GetenvFile - Also checks file
+func (e *Env) GetenvFile(fs vfs.Filesystem, key, def string) string {
```

Comment:
> All that being said, I think an addition like this may benefit from being in a separate PR. It may also be useful to have this moved to a new `env` package so it can be used by other packages (to avoid circular references).

On second thought, I'll take care of this piece myself - don't worry about a separate PR. I'll get you to rebase after I've moved to a new `env` package.